cfripper/model/utils.py

```diff
@@ -101,3 +104,21 @@ def get_managed_policy(managed_policy_arn):
             }
         )
     return None
+
+
+def get_account_id_from_arn(arn) -> Optional[str]:
```

Comment:
Add type information to `arn` parameter here (looks like a `str`)
Same for function below as well :)

Reply:
done